JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -472,6 +474,20 @@
     }
   }
 }
+.sdBinomLS            <- function(N, p, prop = FALSE){
+  if (prop) d <- N else d <- 1
+  
+  sd <- sqrt( N*p*(1-p) )
+  
+  return(sd / d)
+}
+.sdBetaBinomLS            <- function(N, alpha, beta, prop = FALSE){
```

Comment:
I have no idea what this does.

Reply:
computes a standard deviation - sd - for a beta-binomial - BetaBinom - distribution and it's a function from Learn Bayes - LS